index.html

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="ko">
-<head>
+<head> 
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="or:url" content="https://p-anda-market.netlify.app/">
+    <meta property ="og:title" content="판다마켓">
+    <meta property="og:description" content="일상의 모든 물건을 거래해보세요">
+    <meta property="image" content="img/logo_img.png">
```

Comment:
❗️ 수정요청
이러한 og 설정이 제대로 되어 있는지 [확인](https://www.opengraph.xyz/)할 수 있는 방법이 여러가지 있습니다~
확인해보시고 meta 태그를 수정해보세요~
```suggestion
    <meta property="og:image" content="img/logo_img.png">
```